src/main/java/it/pagopa/interop/probing/eservice/operations/repository/query/builder/EserviceContentQueryBuilder.java

```diff
@@ -61,7 +62,7 @@ public Page<EserviceContent> findAllEservicesReadyForPolling(Integer limit, Inte
 
     return new PageImpl<>(pollingActiveEserviceContent,
         PageRequest.of(offset, limit,
-            Sort.by(ProjectConstants.ESERVICES_RECORD_ID_FIELD).ascending()),
+            Sort.by(Eservice_.ESERVICE_RECORD_ID).ascending()),
```

Comment:
does the `Eservice_` model ends with an underscore on purpose?

Reply:
it is authomatically generated by the hibernate-jpamodelgen dependency